<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="2" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="17141" started-at="2021-06-09T17:23:06 IST" finished-at="2021-06-09T17:23:23 IST">
    <groups>
    </groups>
    <test name="USER LOGIN" duration-ms="10351" started-at="2021-06-09T17:23:06 IST" finished-at="2021-06-09T17:23:17 IST">
      <class name="com.Docker.Selenium_Docker.LoginTest">
        <test-method status="PASS" signature="beforeTest(java.lang.String, org.testng.ITestContext)[pri:0, instance:com.Docker.Selenium_Docker.LoginTest@60285225]" name="beforeTest" is-config="true" duration-ms="3181" started-at="2021-06-09T17:23:06 IST" finished-at="2021-06-09T17:23:09 IST">
          <params>
            <param index="0">
              <value is-null="true"/>
            </param>
            <param index="1">
              <value>
                <![CDATA[org.testng.TestRunner@38600b]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeTest -->
        <test-method status="PASS" signature="beforeClass()[pri:0, instance:com.Docker.Selenium_Docker.LoginTest@60285225]" name="beforeClass" is-config="true" duration-ms="44" started-at="2021-06-09T17:23:09 IST" finished-at="2021-06-09T17:23:09 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeClass -->
        <test-method status="FAIL" signature="Login_Test(java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.Docker.Selenium_Docker.LoginTest@60285225]" name="Login_Test" duration-ms="6215" started-at="2021-06-09T17:23:09 IST" finished-at="2021-06-09T17:23:16 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[Inpatient Ward]]>
              </value>
            </param>
            <param index="1">
              <value is-null="true"/>
            </param>
            <param index="2">
              <value is-null="true"/>
            </param>
            <param index="3">
              <value>
                <![CDATA[TRUE]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Current URL not contain 'home.page'
 Actual URL : https://demo.openmrs.org/openmrs/referenceapplication/home.page expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Current URL not contain 'home.page'
 Actual URL : https://demo.openmrs.org/openmrs/referenceapplication/home.page expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at com.Docker.Selenium_Docker.LoginTest.Login_Test(LoginTest.java:35)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Login_Test -->
        <test-method status="PASS" signature="tearDown(org.testng.ITestResult, org.testng.ITestContext, java.lang.reflect.Method)[pri:0, instance:com.Docker.Selenium_Docker.LoginTest@60285225]" name="tearDown" is-config="true" duration-ms="169" started-at="2021-06-09T17:23:16 IST" finished-at="2021-06-09T17:23:16 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=Login_Test status=FAILURE method=LoginTest.Login_Test(java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.Docker.Selenium_Docker.LoginTest@60285225] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[org.testng.TestRunner@38600b]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[public void com.Docker.Selenium_Docker.LoginTest.Login_Test(java.lang.String,java.lang.String,java.lang.String,java.lang.String)]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="afterTest()[pri:0, instance:com.Docker.Selenium_Docker.LoginTest@60285225]" name="afterTest" is-config="true" duration-ms="708" started-at="2021-06-09T17:23:16 IST" finished-at="2021-06-09T17:23:17 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterTest -->
      </class> <!-- com.Docker.Selenium_Docker.LoginTest -->
    </test> <!-- USER LOGIN -->
    <test name="USER LOGIN Invalid User" duration-ms="6767" started-at="2021-06-09T17:23:17 IST" finished-at="2021-06-09T17:23:23 IST">
      <class name="com.Docker.Selenium_Docker.LoginTest">
        <test-method status="PASS" signature="before_Suite()[pri:0, instance:com.Docker.Selenium_Docker.LoginTest@74ad1f1f]" name="before_Suite" is-config="true" duration-ms="10" started-at="2021-06-09T17:23:06 IST" finished-at="2021-06-09T17:23:06 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- before_Suite -->
        <test-method status="PASS" signature="beforeTest(java.lang.String, org.testng.ITestContext)[pri:0, instance:com.Docker.Selenium_Docker.LoginTest@74ad1f1f]" name="beforeTest" is-config="true" duration-ms="1313" started-at="2021-06-09T17:23:17 IST" finished-at="2021-06-09T17:23:18 IST">
          <params>
            <param index="0">
              <value is-null="true"/>
            </param>
            <param index="1">
              <value>
                <![CDATA[org.testng.TestRunner@669d2b1b]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeTest -->
        <test-method status="PASS" signature="beforeClass()[pri:0, instance:com.Docker.Selenium_Docker.LoginTest@74ad1f1f]" name="beforeClass" is-config="true" duration-ms="1" started-at="2021-06-09T17:23:18 IST" finished-at="2021-06-09T17:23:18 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeClass -->
        <test-method status="PASS" signature="Login_Test(java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.Docker.Selenium_Docker.LoginTest@74ad1f1f]" name="Login_Test" duration-ms="4778" started-at="2021-06-09T17:23:18 IST" finished-at="2021-06-09T17:23:23 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[Inpatient Ward]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Adminstrator]]>
              </value>
            </param>
            <param index="2">
              <value is-null="true"/>
            </param>
            <param index="3">
              <value>
                <![CDATA[false]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Login_Test -->
        <test-method status="PASS" signature="tearDown(org.testng.ITestResult, org.testng.ITestContext, java.lang.reflect.Method)[pri:0, instance:com.Docker.Selenium_Docker.LoginTest@74ad1f1f]" name="tearDown" is-config="true" duration-ms="1" started-at="2021-06-09T17:23:23 IST" finished-at="2021-06-09T17:23:23 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=Login_Test status=SUCCESS method=LoginTest.Login_Test(java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.Docker.Selenium_Docker.LoginTest@74ad1f1f] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[org.testng.TestRunner@669d2b1b]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[public void com.Docker.Selenium_Docker.LoginTest.Login_Test(java.lang.String,java.lang.String,java.lang.String,java.lang.String)]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="FAIL" signature="afterTest()[pri:0, instance:com.Docker.Selenium_Docker.LoginTest@74ad1f1f]" name="afterTest" is-config="true" duration-ms="658" started-at="2021-06-09T17:23:23 IST" finished-at="2021-06-09T17:23:23 IST">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The following asserts failed:
Test Name Failed : USER LOGIN Invalid User
Failed while Validating  : Failing of Assertion
Class Failed : com.Docker.Selenium_Docker.LoginTest
 METHOD FAILED  : Login_Test
 IN Line number : 37
Assertion Fail in URL : https://demo.openmrs.org/openmrs/referenceapplication/login.page
 Failure Message  : Why are you failing expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The following asserts failed:
Test Name Failed : USER LOGIN Invalid User
Failed while Validating  : Failing of Assertion
Class Failed : com.Docker.Selenium_Docker.LoginTest
 METHOD FAILED  : Login_Test
 IN Line number : 37
Assertion Fail in URL : https://demo.openmrs.org/openmrs/referenceapplication/login.page
 Failure Message  : Why are you failing expected [true] but found [false]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at resource.BaseTest.afterTest(BaseTest.java:86)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:61)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:366)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:320)
at org.testng.TestRunner.invokeTestConfigurations(TestRunner.java:617)
at org.testng.TestRunner.afterRun(TestRunner.java:847)
at org.testng.TestRunner.run(TestRunner.java:588)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterTest -->
      </class> <!-- com.Docker.Selenium_Docker.LoginTest -->
    </test> <!-- USER LOGIN Invalid User -->
  </suite> <!-- Suite -->
</testng-results>
